tests/cloner_test.go

```diff
@@ -816,7 +821,7 @@ var _ = Describe("Namespace with quota", func() {
 		By("Verify Quota was exceeded in logs")
 		targetPvc, err := utils.WaitForPVC(f.K8sClient, dataVolume.Namespace, dataVolume.Name)
 		Expect(err).ToNot(HaveOccurred())
-		matchString := fmt.Sprintf("\\\"%s-source-pod\\\" is forbidden: exceeded quota: test-quota, requested: requests.cpu=2, used: requests.cpu=0, limited: requests.cpu=1", targetPvc.GetUID())
+		matchString := fmt.Sprintf("\\\"%s-source-pod\\\" is forbidden: exceeded quota: test-quota, requested: limits.memory=536870912,requests.memory=536870912, used: limits.memory=0,requests.memory=0, limited: limits.memory=268435456,requests.memory=268435456", targetPvc.GetUID())
```

Comment:
if openshift changes this string, I assume we need to make the match less exact

Reply:
Yeah, we could make it less strict, maybe just to "requested" that way it can change without issues.

Reply:
Fixed